src/core/media-set/media-set.vala

```diff
@@ -0,0 +1,58 @@
+// This file is part of GNOME Games. License: GPLv3
+
+public class MediaSet : Object {
+	public delegate void MediaCallback (Media media);
+	private const uint HANDLED_MEDIAS_PER_CYCLE = 5;
```

Comment:
I doubt we will get media sets with more than 4 medias hence it will rarely be used at all.

What about either setting it to 1 or just removing it?